docs/UserGuide.adoc

```diff
@@ -1239,18 +1240,21 @@ image::TaskCardDone.png[]
 
 .9+|Task
 |  `*task* add /code MOD_CODE /desc DESCRIPTION [/on DATE] [/at TIME]`
-|  `*task* delete MOD_CODE ID_NUMBER`
-|  `*task* done /code MOD_CODE /id ID_NUMBER`
 |  `*task* edit MOD_CODE ID_NUMBER [/desc DESCRIPTION] [/on DATE] [/at TIME]` +
    `*task* edit MOD_CODE ID_NUMBER [/desc DESCRIPTION] /on non`
+|  `*task* delete MOD_CODE ID_NUMBER`
+//tag::task-summary-second[]
+|  `*task* done /code MOD_CODE /id ID_NUMBER`
 |  `*task* list`
 |  `*task* module /code MOD_CODE`
-|  `*task* not-done`
-|  `*task* find DESCRIPTION [MORE_DESCRIPTIONS]...`
+|  `*task* undone`
+|  `*task* find DESCRIPTION [MORE_DESCRIPTIONS]...` +
 |  `*task* search [/date DATE] [/month MONTH] [/year YEAR]`
+//end::task-summary-second[]
 
```

Comment:
I think should sort by alphabetical order

Reply:
I separated my commands and @NNpanpan's for clearer distinction, and also we can quote the corresponding part in the PPP easier. The commands are organised in terms of logic (CRUD -> queries -> slightly complex queries) so it should be okay.

Reply:
Summary isn't for making sense of commands, but for quick look-up.
If user wants to understand commands, they'd go thru the UG sections, not the summary. So you should keep it alphabetically sorted - consistent with other parts too.

Reply:
Yeah makes sense. I have updated it to be in alphabetical order. Thanks for the feedback!